keyboards/scrabblepad/keymaps/random/keymap.c

```diff
@@ -0,0 +1,75 @@
+#include "scrabblepad.h"
+#include <stdlib.h>
+
+static uint16_t keystroke;
+static bool shift_pressed;
+
+static int lower = (int) KC_A;
```

Comment:
I suggest turning these into preprocessor constants so that you don't have to worry about the type here.
```suggestion
#define LOWER KC_A
```